modules/server/src/main/scala/seqexec/server/gmos/GmosHeader.scala

```diff
@@ -117,7 +117,7 @@ object GmosHeader {
             buildDouble(gmosReader.gratingWavelength, KeywordName.GRWLEN),
             buildDouble(gmosReader.gratingAdjustedWavelength, KeywordName.CENTWAVE),
             buildInt32(gmosReader.gratingOrder, KeywordName.GRORDER),
-            buildDouble(gmosReader.gratingTilt, KeywordName.GRTILT),
+            buildDoublePrecision(gmosReader.gratingTilt, 4, KeywordName.GRTILT),
```

Comment:
Is this the only keyword ever need a specific precission?

Reply:
So far, but I already told Andy we can do this, he surely will come with a list of other keywords.